Client/Frontend/Browser/BrowserViewController/BrowserViewController+Menu.swift

```diff
@@ -29,24 +30,64 @@ extension BrowserViewController {
-    func destinationMenuSection(_ menuController: MenuViewController) -> some View {
-        VStack(spacing: 0) {
-            MenuItemButton(icon: #imageLiteral(resourceName: "menu_bookmarks").template, title: Strings.bookmarksMenuItem) { [unowned self, unowned menuController] in
-                let vc = BookmarksViewController(folder: self.bookmarkManager.lastVisitedFolder(), bookmarkManager: self.bookmarkManager, isPrivateBrowsing: PrivateBrowsingManager.shared.isPrivateBrowsing)
-                vc.toolbarUrlActionsDelegate = self
-                menuController.presentInnerMenu(vc)
-            }
+    func privacyFeaturesMenuSection(_ menuController: MenuViewController) -> some View {
+        VStack(alignment: .leading, spacing: 0) {
+            Spacer(minLength: 15.0)
+            
+            Text(verbatim: Strings.PrivacyFeature.menuSectionTitle.uppercased())
+                .font(.callout)
+                .fontWeight(.semibold)
+                .lineLimit(1)
+                .foregroundColor(Color(.braveLabel)
+            ).padding(.horizontal, 14)
+                        
+            VPNMenuButton(
+                vpnProductInfo: self.vpnProductInfo,
+                description: Strings.PrivacyFeature.braveVPNItemDescription,
+                displayVPNDestination: { [unowned self] vc in
+                    (self.presentedViewController as? MenuViewController)?
+                        .pushInnerMenu(vc)
+                },
+                enableInstalledVPN: { [unowned menuController] in
+                    /// Donate Enable VPN Activity for suggestions
+                    let enableVPNActivity = ActivityShortcutManager.shared.createShortcutActivity(type: .enableBraveVPN)
+                    menuController.userActivity = enableVPNActivity
+                    enableVPNActivity.becomeCurrent()
+                }
+            )
+            
+            // Add Brave Talk and News options only in normal browsing
+            if !PrivateBrowsingManager.shared.isPrivateBrowsing {
+                MenuItemButton(
+                    icon: #imageLiteral(resourceName: "menu-brave-talk").template,
+                    title: "Brave Talk",
+                    subTitle: Strings.PrivacyFeature.braveTalkItemDescription) { [weak self] in
+                    guard let self = self, let url = URL(string: "https://talk.brave.com/") else { return }
+                    
+                    self.popToBVC()
+                    self.finishEditingAndSubmit(url, visitType: .typed)
+                }
+                
+                // Show Brave News if it is first launch and after first launch If the new is enabled
+                if Preferences.General.isFirstLaunch.value || (!Preferences.General.isFirstLaunch.value && Preferences.BraveNews.isEnabled.value) {
+                    MenuItemButton(
+                        icon: #imageLiteral(resourceName: "menu_brave_news").template,
+                        title: "Brave News",
```

Comment:
Do we want to have this localized?

Reply:
See https://github.com/brave/brave-ios/pull/4424#discussion_r741417115

Reply:
Fix at https://github.com/brave/brave-ios/pull/4424/commits/156d50f94bf8089de48e348deca6ec26c6897cdb